monai/bundle/scripts.py

```diff
@@ -1212,14 +1216,8 @@ def _export(
     # add .json extension to all extra files which are always encoded as JSON
     extra_files = {k + ".json": v for k, v in extra_files.items()}
```

Comment:
I would recommand to move the whole `extra_files` logic from the `_export` function to the `ckpt_export` function , since this parameter is unique for the latter one.  You can fix it as a partial function parameter as `include_config_vals`.  Then the saver call in the next line will only need two parameters which looks more consistent to me.

Thanks  
Bin

Reply:
Thank you for your suggestion! This idea will be discussed along with another proposed change in the comments below.